.github/ISSUE_TEMPLATE/provider-issue.md

```diff
@@ -11,29 +11,42 @@ Thank you for opening an issue. Please note that we try to keep the Databricks P
 -->
 
 ### Configuration
+
```

Comment:
Please remove these and other whitespaces between section and comments . The new line gives separation between different section when someone uses the template. If we have white spaces then it makes it a bit confusing on whether to write between the section and comment or after comment.

Reply:
@tanmay-db 
Thank you for your check I removed 
https://github.com/databricks/terraform-provider-databricks/pull/3020/commits/9ce4b43c7e24ab8f623dce5a7e70f9f30f059e78